includes/acl/decompression/impl/transform_animated_track_cache.h

```diff
@@ -655,14 +694,44 @@ namespace acl
 						// Convert to floats and normalize
 						__m128i xyz = _mm_setr_epi32(x, y, z, 0);
 						__m128 xyzf = _mm_cvtepi32_ps(xyz);
+
+#ifdef ACL_PRECISION_BOOST
+
+						rotation_as_vec = _mm_mul_ps(_mm_sub_ps(xyzf, _mm_set_ps1(32767.5F)), _mm_set_ps1(1.0F / 65535.0F));
```

Comment:
For 8 bit quantization we use 1/254, but for 16 bit we use 1/65535? Why the off by 1 for 8 bits but not 16?